README.md

```diff
@@ -320,6 +327,66 @@ Chewy.settings = {
   User.plugin :chewy_observe           # just for User
   ```
 
+#### Example of data request
```

Comment:
I will suggest a little bit distinct order here, with a storytelling like step by step order:

1. Once a record is created (could be done via Rails console), it creates User index too
```ruby
User.create(
  first_name: "test1",
  last_name: "test1",
  email: 'test1@example.com',
  # other fields
)
# UsersIndex::User Import (355.3ms) {:index=>1}
# => #<User id: 1, first_name: "test1", last_name: "test1", email: "test1@example.com", # other fields>
```
2. A query could be exposed at a given `UsersController`:
```ruby
class UsersController < ActionController::Base
  def search
    @users = UsersIndex.query(query_string: { fields: [:first_name, :last_name, :email, ...], query: search_params[:query], default_operator: 'and' })
    render json: @users.to_json, status: :ok
  end
end
```
3. So a request against `http://localhost:3000/users/search?query=test1@example.com` could have a response like
```json
[
  {
    "attributes":{
      "id":"1",
      "first_name":"test1",
      "last_name":"test1",
      "email":"test1@example.com",
      ...
      "_score":0.9808291,
      "_explanation":null
    },
    "_data":{
      "_index":"users",
      "_type":"_doc",
      "_id":"1",
      "_score":0.9808291,
      "_source":{
        "first_name":"test1",
        "last_name":"test1",
        "email":"test1@example.com",
        ...
      }
    }
  }
]
```

WDYT?

Reply:
Thank you! Looks great